src/apps/chifra/pkg/rpcClient/options.go

```diff
@@ -24,3 +29,11 @@ func (o *Options) HasStore() bool {
 
 	return o.Store != nil
 }
+
+func (o *Options) HasStoreWriteable() bool {
```

Comment:
Does `HasWritableStore` make more grammatical sense?

Reply:
Yes! I chose `HasStoreWritable` because it shows in the hints next to `HasStore`, but it looks and sounds awful